### ISSUE TITLE
[Diacritics] Set properly the diacritics offset for all Persian letters

### ISSUE THREAD

**Issue by FrancyCavallari** [vgwb/Antura#235]
![image](https://user-images.githubusercontent.com/17926293/150094526-025bfff9-a2d6-4286-9f2f-46573bd857f3.png)

**Comment by FrancyCavallari** [comment]
@CatmanIta, it seem that persian_dari/DiacriticsComboData is still using Arabic letters and combos, so I cannot set the offsets for new/Persian letters
![image](https://user-images.githubusercontent.com/17926293/150219934-9580dd71-5103-47f3-a877-69ba3bf2646d.png)

**Comment by CatmanIta** [comment]
Yes, we need to refresh it whenever we work with a new language, or letters are added. That table needs to be regenerated each time.
